defaults.yml

```diff
@@ -743,6 +743,78 @@ categories:
             text: Thu 21:46:18
         versions: [Monterey, Big Sur, Catalina]
 
+  - folder: trackpad
+    name: Trackpad
+    description: The trackpad is hardware to control the cursor that comes on all MacBooks.
+    keys:
+      - key: FirstClickThreshold
+        domain: com.apple.AppleMultitouchTrackpad
+        title: Click weight (threshold)
+        description: Choose between Light/Medium/Firm.
+        param:
+          type: int
+          values: [0, 1, 2]
+        # TODO: add default value
```

Comment:
on my Mac, `defaults read com.apple.AppleMultitouchTrackpad FirstClickThreshold` return 1

Reply:
Do you mean to say that is the default value, or is that a value that you had set previously?

Reply:
You're right, `default` means the set value by macOS. Unfortunately that value doesn't show up when you `defaults read`. `defaults delete ${domain} ${key}` should reset the value so that's how I test it.

I tried on a new session on a freshly installed Mac (not a M1 one tho) and `Light` was the default param.

I do not know about how these default values are stored in macOS, hopefully they're the same for every Mac, but I can't affirm that.

Reply:
@yannbertrand I just tested with a Guest account, and confirmed that the default is `Medium`. It was readable through `defaults read`.

`defaults delete` doesn't seem like a good way to deduce the default value. In this case, it was wrong - I think because the value is missing, which somehow translates to a value of `0` for the key and thus rendered as `Light` in the Settings app.

In other words, I think `defaults delete` is more reflective of setting a value of `0`/`false`/`null` than resetting back to the stock macOS default.

Reply:
New PR: #300

Reply:
Actually I just tried on Finder's [`NSTableViewDefaultSizeMode `](https://macos-defaults.com/finder/nstableviewdefaultsizemode.html), `delete`ing this value reset it to `2` "Medium" even tho it doesn't show up in `defaults read NSGlobalDomain`.

So I really don't know how this works 😅